_projects/coala_cli.md

```diff
@@ -1,7 +1,8 @@
 ---
 collaborating_projects:
   - coala
-desc: "There are a number of possible options to improve the coala interface written down in [a draft of cEP-0004](https://github.com/coala/cEPs/pull/29/files)."
+desc: "There are a number of possible options to improve the coala interface written down in
```

Comment:
break this line at 80cpl, and align indented text on next line under first letter of this line.